notebooks/274-Real-ESRGAN-image-restoration/274-Real-ESRGAN-image-restoration.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Construct the command for Model Optimizer.
</pre><p>could you please use Model Optimizer Python API instead of CLI, please check https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html for details</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>